client/app/shared/icon-status.component.js

```diff
@@ -0,0 +1,50 @@
+export const IconStatusComponent = {
+  controllerAs: 'vm',
+  controller: ComponentController,
+  bindings: {
+    status: '<',
+    success: '<?',
+    error: '<?',
+    queued: '<?',
+    inprogress: '<?',
+  },
+  template: `
+   <i class="pficon pficon-ok" ng-if="vm.isSuccess"
```

Comment:
Intentionally verbose, favoring readability rather than conciseness, as was demonstrated above, likely these values are going to change

One below, don't have to worry about translations breaking functionality value and check value(s) are passed by the user, most often specified by the backend for both